wlc/test_base.py

```diff
@@ -63,9 +63,15 @@ def format_body(body):
             return ""
         body = body.decode()
         body = (
-            body.replace(": ", "=").replace("{", "").replace("}", "").replace('"', "")
+            body.replace(": ", "=")
+            .replace("{", "")
+            .replace("}", "")
+            .replace('"', "")
+            .replace("://", "-::")
+            .replace("/", ":")
+            .replace(", ", "--")
+            .replace(" ", "-")
```

Comment:
Please include `[` `]` in the cleanup as well, it is also a problematic char to have in a filename. 

Also, maybe it's time to use regexp here instead of chaining so many replaces.